infoextract.js

```diff
@@ -67,6 +67,30 @@ module.exports = {
 		}
 	},
 
+	getChanneInfoYoutube(channelData) {
+		return new Promise((resolve, reject) => {
```

Comment:
This is an extra promise, just `return` instead of `resolve`. Promises chain, so if you return in one of the `then` or `catch` callbacks, the returned value works its way up the chain.